src/main/java/com/revature/chronicle/services/VideoService.java

```diff
@@ -193,8 +193,9 @@ public boolean deleteVideo(Video video, User user) {
     
     //find a video by it's title 
     public Video findByTitle(String title){
-       
+       System.out.println(title);
```

Comment:
Another reminder to comb through the code for sysouts. You should log instead if you need to.

Reply:
Did you forget to remove this?

Reply:
Yes we did. I just removed it.